### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -6871,7 +6880,7 @@ entired->entered, entire,
 entireity->entirety
 entires->entries
 entirey->entirely
-entirity->entirely
+entirity->entirety, entirely,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: I'd agree with your suggestion and be tempted to drop entirely, it seems like it was influenced by the ones either side of it.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -17144,6 +17162,8 @@ sytntax->syntax
 szenario->scenario
 szenarios->scenarios
 szes->sizes
+szie->size
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add szied equivalent too.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: I'll drop it

**Comment by luzpaz** [comment]
If this is approved please squash it down to one commit. If you want me to to so...then I'd be happy to.

**Comment by luzpaz** [comment]
I added 72c096d last minute. Please also review it. Thanks!

**Comment by peternewman** [comment]
> If this is approved please squash it down to one commit. If you want me to to so...then I'd be happy to.

Squashed and merged as requested. Certainly I've not generally bothered with other commits.

**Pull request merged**